packages/nimble-components/src/wafer-map/tests/data-manager.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { DataManager } from '../modules/data-manager';
+import {
+    Margin,
+    WaferMapColorsScaleMode,
+    WaferMapDie,
+    WaferMapQuadrant
+} from '../types';
+import {
+    getColorsScale,
+    getHighlightedValues,
+    getWaferMapDies
+} from './utilities';
+
+describe('Prerendering module', () => {
+    let dataManagerModule: DataManager;
+    const axisLocation: WaferMapQuadrant = WaferMapQuadrant.topLeft;
+    const canvasDimensions = { width: 100, height: 110 };
+    const dieLabelsSuffix = '%';
+    const dieLabelsHidden = false;
+    const maxCharacters = 3;
+    const defaultMargin: Margin = {
+        top: 20,
+        right: 20,
+        bottom: 20,
+        left: 20
+    };
+
+    beforeEach(() => {
+        dataManagerModule = new DataManager(
+            getWaferMapDies(),
+            axisLocation,
+            canvasDimensions,
+            getColorsScale(),
+            getHighlightedValues(),
+            WaferMapColorsScaleMode.ordinal,
+            dieLabelsHidden,
+            dieLabelsSuffix,
+            maxCharacters
+        );
+    });
+
+    it('should have 60:70 container', () => {
```

Comment:
nit: it's generally cleaner not to include expected values in test names since it's an implementation detail that can change and it doesn't really express the intent of the test. Something like "computes the correct containerDimensions" would be preferable.

Same for other tests in this file.